sdk/js/src/utils.ts

```diff
@@ -86,6 +86,15 @@ export function checkIsServiceWorker(): boolean {
         self.constructor.name === 'ServiceWorkerGlobalScope'
     )
 }
+
+export class UserError extends Error {
+    constructor(error: Error | string) {
+        super(error instanceof Error ? error.message : error)
+        this.name = 'UserError'
+        this.stack = error instanceof Error ? error.stack : undefined
```

Comment:
wont this wipe the original stack for the error?

Reply:
I don't believe so, this is the same implementation as the Node SDK

Reply:
well it looks like its reassigning the stack property to be undefined